src/azure-cli/azure/cli/command_modules/mysql/_help.py

```diff
@@ -183,6 +183,68 @@
         --backup-key $backupKeyIdentifier --backup-identity testBackupIdentity
 """
 
+helps['mysql flexible-server threat-protection update'] = """
+type: command
+short-summary: Update the threat protection settings for a flexible server.
+parameters:
+  - name: --resource-group -g
+    type: string
+    short-summary: Name of the resource group.
+  - name: --server-name -s
+    type: string
+    short-summary: Name of the flexible server.
+  - name: --defender-state
+    type: string
+    short-summary: Determines the state of defender. Valid values are 'Enabled' or 'Disabled'.
+  - name: --dname -dn
+    type: string
+    short-summary: The name of the advanced threat protection resource. The name is required when --defender-state is Enabled.
+  - name: --subscription-id -sid
+    type: string
+    short-summary: The subscription id of the advanced threat protection resource. The subscription id is required when --defender-state is Enabled.
+examples:
+  - name: Update the threat protection settings for a flexible server.
+    text: az mysql flexible-server threat-protection update --resource-group myrg --server-name myserver --defender-state Enabled
+"""
+
+
+helps['mysql flexible-server threat-protection show'] = """
+type: command
+short-summary: Show the threat protection settings for a flexible server.
+parameters:
```

Comment:
Should you also add subscription id as optional parameter?

Reply:
In CLi az login takes care of the subscription id , we don't have to add it explicitly 
Also as per my design doc , it wasn't mandatory , so I have added it as optional param.
But , ya i'll add it in help.py too 
Thanks.